easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-2023.03.0+386-foss-2022b-Java-11-R-4.2.2.eb

```diff
@@ -0,0 +1,100 @@
+easyblock = 'CMakeNinja'
+
+name = 'RStudio-Server'
+version = '2023.03.0+386'
+versionsuffix = '-Java-%(javaver)s-R-%(rver)s'
+local_git_rev = '7872775ebddc40635780ca1ed238934c3345c5de'
+
+homepage = 'https://www.rstudio.com/'
+description = """This is the RStudio Server version.
+RStudio is a set of integrated tools designed to help you be more productive with R.
+
+The server can be started with:
+  rserver --server-daemonize=0 --www-port=8787
+
+If you need a database config one can be created with:
+  MYTMP=`mktemp -d` && echo -e "provider=sqlite\\ndirectory=${MYTMP}/sqlite" > "${MYTMP}/db.conf"
+and then used with:
+  rserver ... --database-config-file="${MYTMP}/db.conf"
+"""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+source_urls = ['https://github.com/rstudio/rstudio/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = [
+    {'v2023.03.0+386.tar.gz': '62a67f3b528dec3957e2ea0b172cedca792e4af6b50542d65d5a53e8ae0901dd'},
+]
+
+builddependencies = [
+    ('ant', '1.10.12', '-Java-%(javaver)s', SYSTEM),
+    ('CMake', '3.24.3'),
+    ('Ninja', '1.11.1'),
+    ('pkgconf', '1.9.3'),
+]
+
+dependencies = [
+    ('Boost', '1.81.0'),
+    ('Java', '11', '', SYSTEM),
+    ('R', '4.2.2'),
+    ('SOCI', '4.0.3'),
+    ('yaml-cpp', '0.7.0'),
+]
+
+osdependencies = [
+    ('pam-devel', 'libpam0g-dev')
+]
+
+# Required until https://github.com/easybuilders/easybuild-framework/pull/4035
+local_start_dir = "%(builddir)s/rstudio-" + version.replace("+", "-")
+
+preconfigopts = " && ".join([
+    # Install dependencies via scripts. Done in subshell to preserve PWD
+    "(export RSTUDIO_TOOLS_ROOT='%(builddir)s'",
+    "cd '%s/dependencies/common'" % local_start_dir,
+    "./install-cef",
+    "./install-dictionaries",
+    "./install-mathjax",
+    "./install-pandoc",
+    "./install-packages",
+    "./install-npm-dependencies)",
+    ""
+])
+
+configopts = " ".join([
+    "-DRSTUDIO_TOOLS_ROOT='%(builddir)s'",
+    "-DRSTUDIO_TARGET=Server",
+    "-DRSTUDIO_USE_SYSTEM_BOOST=ON",
+    "-DRSTUDIO_USE_SYSTEM_SOCI=ON",
+    "-DRSTUDIO_USE_SYSTEM_YAML_CPP=ON",
+    "-DQUARTO_ENABLED=OFF",  # Not available on all archs, use pandoc fallback
```

Comment:
In https://github.com/easybuilders/easybuild-easyconfigs/pull/15524/files#diff-71ad7363a3fabd15a4e213d910c9a091417d9e9672ae0430a4156ca95818cce6R36 I think I had an approach that allowed you to have Quarto on the platforms that supported it.

Reply:
Actually, looking at https://github.com/rstudio/rstudio/blob/8a99de056035023c478eb079ca4a4a50293e9b69/dependencies/common/install-quarto#L66 I think Quarto is now supported for all the platforms we are interested in?

Reply:
I tried commenting that line out, and it failed at the `configure` stage.